instagram/src/components/PostContainer/Post.js

```diff
@@ -7,7 +7,8 @@ class Post extends React.Component{
     constructor(props){
         super(props);
         this.state={
-            likes: props.post.likes
+            likes: props.post.likes,
```

Comment:
avoid initializing the state with props! This can cause issues down the line with the child component not rerendering upon parent changes in state.